charts/stable/synapse/templates/_configmap.tpl

```diff
@@ -91,17 +91,16 @@ data:
 
     {{- if .Values.matrix.registration.require3PID }}
     registrations_require_3pid:
-        {{- range .Values.matrix.registration.requires3PID }}
-        - {{ . | .medium }}
+        {{- range .Values.matrix.registration.require3PID }}
+        {{ print "- " (.medium | quote) }}
         {{- end }}
```

Comment:
Is there a way to evaluate questions.yaml? I seems like it's not producing the expected output here and it gives an error about `'.medium'` not existing. Looks like it's probably being passed an empty object as the `.` variable.

I was able to register with email without the `requires3PID` by setting the `allowedLocal3PIDs` instead so some progress.

Reply:
No, there is none.
Also it's not questions.yaml that is throwing that, questions.yaml just creates a values.yaml file internally.

Reply:
Yeah, I realize that. When I created a values.yaml things rendered fine. I've either for a typo in questions.yaml or  it's outputting a different structure than I am expecting it to.